packages/onboarding-ui/src/common/OnboardingLogo.tsx

```diff
@@ -1,13 +1,15 @@
 import { Box } from '@rocket.chat/fuselage';
-import { RocketChatLogo } from '@rocket.chat/logo';
+import { TaggedRocketChatLogo, RocketChatLogo } from '@rocket.chat/logo';
 import type { ReactElement } from 'react';
 
-export const OnboardingLogo = ({ tag }: { tag?: string }): ReactElement => (
+export const OnboardingLogo = (): ReactElement => (
   <Box width='100%' maxWidth={224}>
-    <RocketChatLogo tagTitle={tag} />
+    <RocketChatLogo />
   </Box>
 );
 
 export const OnboardingLogoCloud = (): ReactElement => (
-  <OnboardingLogo tag='CLOUD' />
+  <Box width='100%' maxWidth={224}>
+    <TaggedRocketChatLogo tagTitle='CLOUD' />
```

Comment:
We can't to maintain the tag default in the cloud logo.

Reply:
You're saying that because we're not using the tag anymore right? So could we just replace in the page for the `OnboardingLogo`?